azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf

```diff
@@ -0,0 +1,96 @@
+variable "express-azure-functions" {
+  default = {
+    repository = {
+      organization    = "pagopa"
+      name            = "io-functions-express"
+      branch_name     = "master"
+      pipelines_path  = ".devops"
+      yml_prefix_name = null
+    }
+    pipeline = {
+      enable_code_review = true
+      enable_deploy      = true
+    }
+  }
+}
+
+locals {
+  # global vars
+  express-azure-functions-variables = {
+    cache_version_id = "v1"
+  }
+  # global secrets
+  express-azure-functions-variables_secret = {
+
+  }
+  # code_review vars
+  express-azure-functions-variables_code_review = {
+
+  }
+  # code_review secrets
+  express-azure-functions-variables_secret_code_review = {
+    danger_github_api_token = module.secrets.values["DANGER-GITHUB-API-TOKEN"].value
```

Comment:
do u use DANGER?

Reply:
yes, but thanks for pointing me this condition to check